#!/bin/sh /usr/share/dpatch/dpatch-run
## 314-magic-add-lzma.dpatch by Robert Millan <rmh@aybabtu.com>
##
## DP: New magic for the LZMA compression format (Closes: #364260).

@DPATCH@

diff -Naur file-4.20.orig/magic/Magdir/compress file-4.20/magic/Magdir/compress
--- file-4.20.orig/magic/Magdir/compress	2007-04-03 13:10:23.000000000 +0000
+++ file-4.20/magic/Magdir/compress	2007-04-03 13:15:02.000000000 +0000
@@ -174,6 +174,11 @@
 >6	byte		x			version %d
 >7	byte		x			\b.%d
 
+# Type: LZMA
+# URL:  http://www.7-zip.org/sdk.html
+# From: Robert Millan <rmh@aybabtu.com>
+0	string		]\000\000\200\000\377\377\377\377\377\377\377\377	LZMA compressed data
+
 # AFX compressed files (Wolfram Kleff)
 2	string		-afx-		AFX compressed file data
 
